datafusion/physical-expr/src/aggregate/first_last.rs

```diff
@@ -63,9 +64,15 @@ impl FirstValue {
             expr,
             ordering_req,
             requirement_satisfied,
+            ignore_null: false,
         }
     }
 
+    pub fn ignore_null(mut self, ignore_null: bool) -> Self {
```

Comment:
For a builder style API, a common API name pattern should be (e.g., the existing `with_requirement_satisfied`):

```suggestion
    pub fn with_ignore_null(mut self, ignore_null: bool) -> Self {
```

Reply:
Changed. Thanks!